src/api/__tests__/Annotations.test.js

```diff
@@ -100,6 +101,38 @@ describe('api/Annotations', () => {
         });
     });
 
+    describe('updateAnnotation()', () => {
+        const errorCallback = jest.fn();
```

Comment:
Let's create these mock functions in each test so we're not reusing the same ones. This was a mistake on my part when I wrote the create/fetch tests.